### PR TITLE
Hompage tests

### PR DESCRIPTION
**Pull request by martypowell**
Added tests for the new stories

### CODE REVIEW

File: spec/features/homepage/homepage.automater.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            'september',
+            'october',
+            'november',
+            'december'].includes(month.toLowerCase());
+    }
+
+    function isValidDay(dayStr) {
+        try {
+            const day = parseInt(dayStr);
+            return day > 0 && day <= 31;
```
Comment (mxsnyder) [inline spec/features/homepage/homepage.automater.js]: It looks like February 30 would be considered a valid day. I think you might want to combine this and isValidMonth and use Date.parse, checking to see if the result is a number or NaN.

File: spec/features/homepage/homepage.automater.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            });
+
+            return dates.length === validDates.length;
+        }
+        catch (ex) {
+            return false;
+        }
+    }
+
+    function isValidMonth(month) {
```
Comment (mxsnyder) [inline spec/features/homepage/homepage.automater.js]: This and isValidDay should be in their own object. Also, will the month ever be abbreviated? Check out Date.parse().

**Review by mxsnyder — CHANGES_REQUESTED** [review]
Overall, I really like what you've done here!

File: spec/features/homepage/homepage.automater.js (hunk 2 above)
Comment (martypowell) [reply spec/features/homepage/homepage.automater.js]: done

File: spec/features/homepage/homepage.automater.js (hunk 1 above)
Comment (martypowell) [reply spec/features/homepage/homepage.automater.js]: done

**Comment by martypowell** [comment]
@mxsnyder @ascire this is really ready for a review this time.

**Review by mxsnyder — APPROVED** [review]
While I'm OK with this, it bugs me a little that the use of arrow functions, named functions, and function variables isn't consistent across files. Not the end of the world, but you should consider picking one and sticking with it.

**Comment by martypowell** [comment]
Closing this pull request, to open a fresh es6 one.

**Pull request closed without merging**